File: Java/Iniciante/1013.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  
    public static void main(String[] args) throws IOException {
      Scanner scanner = new Scanner(System.in);

      int a = scanner.nextInt();
      int b = scanner.nextInt();
      int c = scanner.nextInt();
      int maiorAB = (a + b + Math.abs(a-b)) / 2;
      int maiorABC = (maiorAB + c + Math.abs(maiorAB - c)) / 2;
        
      String resultado = String.format("%s eh o maior", maiorABC);
      System.out.println(resultado);
    }
}

File: PHP/Iniciante/1013.php
<?php

list($a, $b, $c) = explode(" ", readline());

$a = intval($a);
$b = intval($b);
$c = intval($c);

$maiorAB = ($a + $b + abs($a-$b)) / 2;
$maiorABC = ($maiorAB + $c + abs($maiorAB-$c)) / 2;

echo "$maiorABC eh o maior\n";
  
?>
